params.md

```diff
@@ -0,0 +1,77 @@
+The [draft Attribution Reporting API
```

Comment:
Can we name this file to chromium-params.md, and maybe put it in a separate directory.